.cspell.json

```diff
@@ -174,7 +174,10 @@
       "hrefs",
       "DEBEZIUM",
       "Debezium",
-      "debezium"
+      "debezium",
+      "dicom",
+      "qido",
+      "wado"
```

Comment:
Can you please add these words to the ignorewords of this file?

Reply:
ok, but these are ok or should I remove them from here?

Reply:
Words moved down to ignorewords